### DISCUSSION TITLE
Anfrage bezüglich Lizenz

### DISCUSSION THREAD

**Discussion by Sn0w3y** [tbnobody/OpenDTU#1340] [General]
Hallo zusammen, 

speziell Hallo @tbnobody - ich habe eine Implementierung zu openEMS geschrieben (über die API von openDTU).

Kannst du mich dazu eventuell irgendwie kontaktieren? Ich hätte fragen bezüglich der GPL Lizenz :)

Meine Frage dazu wäre, ob es in Ordnung für dich ist, wenn ich (Chamer) diese Implementierung in der openEMS Community veröffentlichen darf und somit deine API nutzen darf, um die WR auszulesen und steuern zu können :)

Grüße von der Obpf. in die Obpf. :)

**Comment by tbnobody** [comment]
Hallo @Sn0w3y,

verwendest du die Hoymiles Bibliothek in deiner Implementierung oder ist der Ablauf so, dass deine Implementierung nur auf die WebAPI zugreift die weiterhin von einem ESP bereitgestellt wird? Ein API Zugriff ist meiner Meinung nach komplett unkritisch. 
Es gibt ja auch z.B. schon den ioBroker Adapter für OpenDTU: https://github.com/o0shojo0o/ioBroker.opendtu

Ich frage mich nur gerade ob es irgendwie Sinn macht in den Discussions einen Announcement Thread zu erstellen in dem sich Maintainer externer Erweiterungen subscriben können falls ich doch mal größere Breaking Changes an der API (Web oder MQTT) machen muss. Im Optimalfall würde ich das da vor dem Release entsprechend ankündigen damit ihr (cc @o0shojo0o) ggf. was anpassen könnt und nicht überrumpelt werdet.

**Reply by Sn0w3y** [reply]
Wäre super ! Dann lass uns das so machen!

**Reply by o0shojo0o** [reply]
Ja das wäre eine tolle Sache 👍

**Reply by tbnobody** [reply]
Habe unter #1463 sowas angefangen...

**Reply by Sn0w3y** [reply]
Subscribed ! :)

**Comment by Sn0w3y** [comment]
https://community.openems.io/t/opendtu-implementierung/1928/8

Hier der Link zum Thread in openEMS :)
